app/models/manageiq/providers/redhat/infra_manager/external_distributed_virtual_switch.rb

```diff
@@ -0,0 +1,2 @@
+class ManageIQ::Providers::Redhat::InfraManager::ExternalDistributedVirtualSwitch < ManageIQ::Providers::InfraManager::ExternalDistributedVirtualSwitch
```

Comment:
I don't think we need an external_distributed_virtual_switch at core, you can just inherit from `ManageIQ::Providers::InfraManager::DistributedVirtualSwitch` here